cljfmt/src/cljfmt/main.clj

```diff
@@ -19,8 +19,8 @@
     (apply println args)))
 
 (defn- relative-path [dir file]
```

Comment:
`[^File dir ^File file]` tends to be more DRY / future-proof than the current hinting, since otherwise a future change could introdcue a new `dir` reference that is not type-hinted.

Reply:
This seems like a good change to make.

Reply:
done